shared/src/crypto.rs

```diff
@@ -414,6 +449,15 @@ fn x509_date_valid(cert: &X509) -> Result<bool, ErrorStack> {
     return Ok(expirydate > now && now > startdate);
 }
 
+pub fn load_certificates_from_file(ca_file: PathBuf) -> Result<X509,SamplyBeamError> {
+    let file = ca_file.as_path();
+    let content = std::fs::read(file)
```

Comment:
Why not the async function from tokio? Would this bloat the binary?

Reply:
What is the benefit if we block immediately?